karma.conf.js

```diff
@@ -87,7 +93,14 @@ module.exports = function(config) {
 						]
 					}
 				],
-  				noParse: [
+				postLoaders: [
+					{ //delays coverage til after tests are run, fixing transpiled source coverage error
+						test: /\.js$/,
+						exclude: /(test|node_modules|bower_components)\//,
+						loader: 'istanbul-instrumenter'
```

Comment:
Do we need to add this to package.json? https://www.npmjs.com/package/istanbul-instrumenter-loader